components/contribution-flow/utils.js

```diff
@@ -249,7 +227,7 @@ export const generatePaymentMethodOptions = (
 
       uniquePMs.push({
         key: STRIPE_PAYMENT_ELEMENT_KEY,
-        title: <FormattedMessage defaultMessage="Bank debit" />,
+        title: <FormattedMessage defaultMessage="Bank debit (automated)" />,
```

Comment:
IMO, we can get rid of the subtitle and use SEPA or ACH as the title. But this is only my opinion, I'm OK with this solution as well.

Reply:
Like "Bank debit (SEPA)" or "Bank debit (ACH)"? Could work.